algrtm/src/components/shared/CustomLink.js

```diff
@@ -3,22 +3,27 @@ import Icon from "./Icon";
 
 const CustomLink = ({ text, href, variant, icon, alt }) => {
   let linkClass =
-    "flex justify-center items-center select-none text-sm font-medium font-semibold px-5 py-2 h-10 rounded-md transition ease-in-out duration-300 animationSmall";
+    "flex justify-center items-center select-none text-sm font-medium font-semibold px-5 py-2 h-[2.75rem] rounded-md transition ease-in-out duration-300 animationSmall";
 
   if (variant === "primary") {
     linkClass +=
       " border border-neutralLight-neutral40 bg-neutralLight-neutral10 text-neutralLight-neutral100";
   } else if (variant === "secondary") {
     linkClass +=
       " border border-neutralLight-neutral80 bg-neutralLight-neutral100 text-neutralLight-neutral10";
+  } else if (variant === "tertiary") {
+    linkClass +=
+      "bg-neutralLight-neutral100 text-neutralLight-neutral10 pl-0 pr-0";
```

Comment:
The space before the text is missing, so the strings join together making the animation not work
![image](https://github.com/Nospoko/algrtm/assets/26410548/21b11cc2-3521-44e2-8ca2-c84f05ccf336)